moveit_core/online_signal_smoothing/include/moveit/online_signal_smoothing/smoothing_base_class.h

```diff
@@ -74,13 +76,15 @@ class SmoothingBaseClass
    * @param position_vector array of joint position commands
    * @return True if initialization was successful
    */
-  virtual bool doSmoothing(std::vector<double>& position_vector) = 0;
+  virtual bool doSmoothing(Eigen::VectorXd& positions, Eigen::VectorXd& velocities, Eigen::VectorXd& accelerations) = 0;
 
   /**
    * Reset to a given joint state
    * @param joint_positions reset the filters to these joint positions
    * @return True if reset was successful
    */
-  virtual bool reset(const std::vector<double>& joint_positions) = 0;
+  virtual bool reset(const Eigen::VectorXd& positions, const Eigen::VectorXd& velocities,
```

Comment:
Same here, add velocities and accelerations to docstring

Reply:
done